src/models/ModelInterface.jl

```diff
@@ -75,20 +71,26 @@ struct ImplementationNotes
 end
 
 """
-`implements(m::Model, tag::ScopeTag) -> Union{ImplementationNotes, Nothing}`
+`implements(m::MyModel, tag::ScopeTag) -> Union{ImplementationNotes, Nothing}`
 
 If `m` implements the GATSegment referred to by `tag`, then return the
 corresponding implementation notes.
 """
 implements(m::Module, ::Type{Val{tag}}) where {tag} = nothing
 
-implements(m::Model, tag::ScopeTag) = implements(m, Val{tag})
+implements(m, tag::ScopeTag) = implements(m, Val{tag})
+
+impl_type(m, tag::ScopeTag) = impl_type(m, Val{tag})
 
-implements(m::Model, theory_module::Module) =
+impl_type(m, mod::Module, name::Symbol) = 
+  impl_type(m, gettag(ident(mod.Meta.theory; name)))
```

Comment:
Remind me what the `;name` without a kwarg does?

Reply:
it's sugar for `name=name` as a keyword argument